ERCS/erc-4337.md

```diff
@@ -583,13 +531,11 @@ The result `SHOULD` be set to the **userOpHash** if and only if the request pass
   * **code: -32503** - UserOperation out of time-range: either wallet or paymaster returned a time-range, and it has already expired (or will expire soon)
     * The `data` field SHOULD contain the `validUntil` and `validAfter` values
     * The `data` field SHOULD contain a `paymaster` value, if this error was triggered by the paymaster
-  * **code: -32504** - transaction rejected because paymaster (or signature aggregator) is throttled/banned
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
-  * **code: -32505** - transaction rejected because paymaster (or signature aggregator) stake or unstake-delay is too low
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
+  * **code: -32504** - transaction rejected because paymaster is throttled/banned
```

Comment:
irrelevant, as it is removed to an external ERC

Reply:
Removed all mentions of error codes.